package/setup.py

```diff
@@ -303,7 +303,7 @@ def extensions(config):
         cpp_extra_compile_args.append('-stdlib=libc++')
         cpp_extra_compile_args.append('-mmacosx-version-min=10.9')
         cpp_extra_link_args.append('-stdlib=libc++')
-        cpp_extra_link_args.append('-mmacosx-version-min=10.7')
+        cpp_extra_link_args.append('-mmacosx-version-min=10.9')
```

Comment:
does this bump the minimum required macos?

Reply:
I don't think it does since we were using a minimum compile version of 10.9 anyways [line 304].

In any case:
1) Mavericks was released in 2013 and was EOL in 2016.
2) Python 3.9 only supports 10.9+ anyways.